lib/extensions.js

```diff
@@ -162,10 +162,23 @@ module.exports = function(realmConstructor, context) {
             Object.keys(this)
                 .concat(Object.keys(Object.getPrototypeOf(this)))
                 .forEach(key => {
-                    const value = this[key];
-                    result[key] = value instanceof realmConstructor.Object || value instanceof realmConstructor.Collection
-                        ? value.toJSON(key, cache)
-                        : value;
+                    const org = this[key];
```

Comment:
I personally prefer not to abbreviate local variables like this - I'll let you decide if you want this suggestion.

```suggestion
                    const original = this[key];
```

Reply:
Agreed. After creating the PR I gave it a workover.